engines/terraform/platform.go

```diff
@@ -50,15 +50,15 @@ type Plugin struct {
 func (p PlatformSpec) GetLibrary(name string) (*Library, error) {
 	library, ok := p.Libraries[name]
 	if !ok {
-		return nil, fmt.Errorf("library %s not found in platform spec", name)
+		return nil, fmt.Errorf("library %s not found in platform spec, configured libraries in platform are: %v", name, slices.Collect(maps.Keys(p.Libraries)))
```

Comment:
```suggestion
		return nil, fmt.Errorf("plugin library %s not found in platform spec, configured libraries in platform are: %v", name, slices.Collect(maps.Keys(p.Libraries)))
```

What does this print when no libraries are configured at all?

Reply:
Good question, its worth a test

Reply:
Added a commit after testing this. The error now reads like this:
failed to resolve library for plugin suga/aws-lambda, library suga not found in platform spec, configured libraries in platform are: []
I think that's pretty clear for now